<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="food_table">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>361</width>
     <height>571</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Food ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="this_food_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>351</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Food ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>610</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>THIS RESERVATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pricelcdNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total price</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="depositlcdNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
